src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -1896,6 +1905,25 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Return False
         End Function
 
+        Public Overrides Function IsParameterNameXmlElementSyntax(node As SyntaxNode) As Boolean Implements ISyntaxFacts.IsParameterNameXmlElementSyntax
+
+            Dim xmlElement = TryCast(node, XmlElementSyntax)
+            If xmlElement IsNot Nothing Then
+                Return xmlElement.StartTag.Name.ToString() = DocumentationCommentXmlNames.ParameterElementName
```

Comment:
should not need TOstring() here (same for c# if you do this there).

Reply:
How? `Name` is an XmlNodeSyntax.

Reply:
well.. check that its' the right type, decompose it, eventually get to a token, and check .ValueText on that :)

Reply:
Fixed, thanks. For some reason C# didn't require the extra type check.